drivers/modem/ublox-sara-r4.c

```diff
@@ -853,6 +957,15 @@ static void modem_reset(void)
 		goto restart;
 	}
 
+	/* configure UBlox LOCATION FINDER service */
+	ret = modem_cmd_send(&mctx.iface, &mctx.cmd_handler,
+			     NULL, 0, "AT+ULOCCELL=1",
```

Comment:
Just for my information, where did you find ULOCCELL At command documentation for SARA R4. I didn't find any. I only found AT+ULOC. Thanks for your help

Reply:
It is part of the U2 intergation guide:
https://www.u-blox.com/sites/default/files/SARA-G3-U2_SysIntegrManual_%28UBX-13000995%29.pdf

Reply:
There is also reference to r4 compatibility in another ublox doc: UBX16029218-R13

The cell locator command isn't compatible with the old versions.

Reply:
ok thanks for this explanation